CONTRIBUTING.md

```diff
@@ -50,3 +50,10 @@ Examples should be written in valid Swift snippets and not written in pseudo-cod
 ## Updating table of contents
 
 There should be no links to empty pages from either the [README.md](README.md) file or the category table of content pages. Empty pages may exist in the repository as long as no active page links to it. When adding a new convention, the contributor is responsible for adding links in the category table of contents as well as to [README.md](README.md).
+
+
+## Test changes in GitHub Pages locally with Jekyll
+
+Please refer to [Testing Your Github Pages site locally with Jekyll](https://help.github.com/en/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll).
```

Comment:
Can we put a very basic "best case" set of instructions? like:

cd into the repository
` $ cd /path/to/swift-guide`
Install the required libraries
`$ bundle install`
Run the server
`$ bundle exec jekyll serve`

For details on how to get the bundler installed, see please refer to [Testing Your Github Pages site locally with Jekyll](https://help.github.com/en/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll).

Reply:
That makes sense. I'll also mention the pre-requisites to run such commands such as Ruby 2.5+ and Bundler.